src/lib/hooks/useServiceWorker.js

```diff
@@ -0,0 +1,50 @@
+// @flow
```

Comment:
This file is a duplicate of _src/lib/utils/useServiceWorker.js_, let's remove that one and keep this coming from master

Reply:
@mauriguz 

okay. then please:
1. remove hook copy from src/lib/utils 
2. place the latest version from master in  src/lib/hooks

Reply:
fixed